awslabs/smithy-language-server
# Add WebSocket support and launcher

*Description of changes:*
- Add websocket support for LSP, such that using the LSP over websockets is possible

*Testing:*
- Launch WSS via `./gradlew run --args="3000 --ws"`, confirm a websocket client can connect and communicate with it
- Launch stdin/stdout via `./gradlew run --args="0"`, confirm no behavior change
- Bump version, Publish to maven local, update vscode extension version, confirm no behavior change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
